reference/OpenAPI.json

```diff
@@ -71851,6 +72106,54 @@
             "description": "Filter customers by the name property.",
             "name": "name"
           },
+          {
```

Comment:
Nie wiem czy to nie powinno być zdefiniowane w `#/components/parameters/` i później używać tego przez $ref. Przynajmniej ja tak do tego podchodzę w products. Możemy to przedyskutowac.

Reply:
Nie mamy na to systemu, nie mamy nigdzie tego opisanego. Brak receptury pisania nazw itp, chce to zostawić narazie. Ustalmy jak to robić i można przez to przejść i to pozmieniać w wiekszej ilości miejsc.